src/components/calcite-flow-item/calcite-flow-item.e2e.ts

```diff
@@ -180,15 +180,20 @@ describe("calcite-flow-item", () => {
     expect(backButtonNew).not.toBeNull();
 
     expect(await backButtonNew.isVisible()).toBe(true);
+  });
+
+  it("Back method should remove element", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent("<calcite-flow-item></calcite-flow-item>");
+
+    let element = await page.find("calcite-flow-item");
 
-    const eventSpy = await page.spyOnEvent("calciteFlowItemBackClick", "window");
+    await element.callMethod("back");
```

Comment:
API-wise, would it be useful for `back` to return the removed element?

Reply:
yup